interfaces/builtin/broadcom_asic_control.go

```diff
@@ -49,9 +40,11 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 `
 
 const broadcomAsicControlConnectedPlugUDev = `
-KERNEL=="linux-user-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-kernel-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-bcm-knet", TAG+="###SLOT_SECURITY_TAGS###"
+# This file contains udev rules for accessing broadcom asic kernel module.
+#
+# Do not edit this file, it will be overwritten on updates
+
+SUBSYSTEM=="net", KERNEL=="bdev", TAG+="%[1]s"
```

Comment:
@jdstrand, I took a closer look today on a machine a kernel built for openswitch. 

A udev rule(/etc/udev/rules.d/60-bcm-driver.rules) executes external comment to creates three character device files(via mknod) by using loadable kernel modules, However, I am not able to query the information for all three character devices on the machine.
http://paste.ubuntu.com/25262959/

I found some hints after exporting the content of the udev database (keyword: "linux-kernel-bde" and "bcm0")
http://paste.ubuntu.com/25262850/
Then query all information for the first matched device
http://paste.ubuntu.com/25262846/
Meanwhile, there's another 'bcm0' device can be found in sysfs (the second matched device)
http://paste.ubuntu.com/25262841/

Based on these findings, the udev tagging rule for this interface would be

```
SUBSYSTEM=="pci", DRIVER=="linux-kernel-bde", TAG+="%[1]s"
SUBSYSTEM=="net", KERNEL=="bcm[0-9]*", TAG+="%[1]s"
```


Also apparmor policy is needed to update somehow by adding a few lines, here are what I have in mind.
```
  /sys/devices/pci[0-9]*/**/config r,
  /sys/devices/pci[0-9]*/**/{,subsystem_}device r,
  /sys/devices/pci[0-9]*/**/{,subsystem_}vendor r,

  /sys/bus/pci/devices/** r,
  /run/udev/data/+pci:[0-9]* r,

```